package.json

```diff
@@ -46,7 +46,7 @@
   ],
   "scripts": {
     "prepublishOnly": "npm run build",
-    "build": "tsup src/index.ts --format esm,cjs --dts --clean",
+    "build": "tsup src/index.ts --format esm,cjs --dts --clean && cp dist/index.d.ts dist/index.d.cts",
```

Comment:
Unfortunately, this is not really portable since windows does not support "cp". I'll think of something.

Reply:
A simple solution would be to use smth like: https://github.com/sindresorhus/cpy-cli

Reply:
I already got something using node -e ...